catalyst/data/utils.py

```diff
@@ -0,0 +1,33 @@
+from typing import List, Union
+
+from torch import int as tint, long, short, Tensor
+
+
+def process_labels(labels: Union[Tensor, List[int]]) -> List[int]:
```

Comment:
as far as this is function on top of `labels` name like `somethin_labels` sounds great
btw, could we name it to `convert_labels2list`? because, it looks like it's exactly what this function is doing :)